README.md

```diff
@@ -114,7 +114,7 @@ Before rendering real content, both the datafile and the user must be available
 
 #### Load the datafile synchronously
 
-Synchronous loading is the preferred method to ensure that Optimizely is always ready and doesn't add any delay or asynchronous complexity to your application.
+Synchronous loading is the preferred method to ensure that Optimizely is always ready and doesn't add any delay or asynchronous complexity to your application. When initializing with both the SDK key and datafile, the SDK will use the given datafile to start, then download the latest version of the datafile in the background.

```

Comment:
so just one thing, could you also update const optimizelyClient = createInstance({
  datafile: window.datafile,
});

to include the SDK key?

Reply:
This is done